src/picterra/__main__.py

```diff
@@ -19,8 +19,19 @@
 
 
 def parse_args(args):
+    # Instantiate client, just for checking URl and api key
+    try:
+        client = APIClient()
```

Comment:
Wouldn't it be cleaner to just add an 'apikey' option to the cli that would print the api key ?

I feel we are a bit reinventing the wheel given your api key is in `PICTERRA_API_KEY`. Also, keep in mind users aren't like us juggling with test and production API keys - they have one so its unlikely it will be setup wrong, no ? So yeah not sure this whole thing is worth it to be honest. What is it solving ?

Reply:
Checked (gor rid of all)